### ISSUE TITLE
Create github issue template

### ISSUE THREAD

**Issue by Guria** [umputun/remark#176]
Some bugs created by people visting radio-t website. Though it could br already fixed in development vetsion.
We can instruct users to check remark42.com/demo first.

**Comment by paskal** [comment]
Things I thought of for PR template:

- [ ] Readme updated if these changes add new functionality or change existing one
- [ ] Parameters section in the doc is updated if CLI parameters are changed

Also, I thought that we sometimes add new CLI options but forget adding them to the Parameters section in the Readme: it should be easy to write a CI step which will compare the output of `--help` run with Readme and check if something is missing.

**Comment by umputun** [comment]
I think the ticket is about the issue template, not PR template. I'm not sure what the issue template could be and have serious doubts if we even need one. Personally, I don't think issue templates that surround this process with rules and formalities. Most of the tickets we got were just fine without any special templates.

Regarding PR template - not sure either. Maybe we just need a contribution document describing our expectations. I don't think adding any automation to verify automatically a lack of readme updates have enough value. This is currently handled by the reviewer and I don't think this part really needs any automation.

**Comment by paskal** [comment]
I would agree on the issue template as I've never seen anything useful: experienced reporters will write everything you expect to know in the first message, and you'll need to ask inexperienced reporters for details despite having the issue template.

Regarding the PR template - I'm OK with not having it and just writing a thing or two to the "Contributions" section in the readme, however, I don't see how automation of parameters documentation check would hurt anybody. I've seen and fixed missing documentation params at least once and I would prefer to rely on robots rather than on people for reporting this problem.

I can agree that that's insignificant and maybe not worth time implementing, but I assume that I can write a bash script for it within 30 minutes, and the reviewer will have one thing less to think about.
